tests/unit/jobs/ContentJobsTest.php

```diff
@@ -737,4 +737,25 @@ public function itShouldForceDeleteOneContent()
         $this->assertNull(Content::find($first->id));
         $this->assertNotNull(Content::withTrashed()->find($second->id));
     }
+
+
+    /** @test */
+    public function cantUpdateContentWithNonExistingThumb()
+    {
+        $content = $this->tester->haveContent([
```

Comment:
I think that we can fit this on one line since it's only a type, also bellow there are two empty lines instead of one :)